app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
I see. This is a very old comment and I expect it to be rather obsolete now. Licensing the whole folder feels wrong. Instead, we should add the license to all copies files header.